WebRoot/WEB-INF/tags/infra/skinAndRedirect.tag

```diff
@@ -28,6 +29,10 @@
 		PageContext pageContext = (PageContext)jspContext;
 		String skin = (String)pageContext.findAttribute("skin");
 
+		if (!Pattern.matches("^[0-9A-Za-z]*$", skin)) {
```

Comment:
Shouldn't this be a `+` instead of a `*`?

Reply:
Yes, you are right. * expands to "zero or more times", while we need "one or more times", exclusing empty string. I will fix the PR now.

Reply:
Check this one please @mss